Solutions/AristaAwakeSecurity/Analytic Rules/ModelMatchesWithMultipleDestinationsByDevice.yaml

```diff
@@ -0,0 +1,62 @@
+id: dfa3ec92-bdae-410f-b675-fe1814e4d43e
+name: Awake Security - Model Matches With Multiple Destinations By Device
+description: This query searches for devices with multiple possibly malicious destinations.
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AristaAwakeSecurity
+    dataTypes:
+      - AwakeSecurity
```

Comment:
If the detection or hunting query operates on a Kusto Function/ Parser instead of the table (like Syslog, CommonEventFormat, _CL), dataTypes should be the Kusto Function name / Parser name and not the table name. However, looking at the query it seems that we are using just the table CommonSecurityLog and not some parser. Please correct me if I am mistaken here. If we are just using the table the dataTypes value should generally be the tablename i.e : CommonSecurityLog instead of AwakeSecurity